js_modules/dagster-ui/packages/ui-core/src/assets/__fixtures__/AssetGroupTags.fixtures.ts

```diff
@@ -0,0 +1,147 @@
+import {MockedResponse} from '@apollo/client/testing';
+
+import {buildAssetNode, buildAutoMaterializePolicy, buildSensor} from '../../graphql/types';
+import {DUNDER_REPO_NAME} from '../../workspace/buildRepoAddress';
+import {ASSET_GROUP_METADATA_QUERY} from '../AssetGroupRoot';
+import {
+  AssetGroupMetadataQuery,
+  AssetGroupMetadataQueryVariables,
+} from '../types/AssetGroupRoot.types';
+
+export const GROUP_NAME = 'my_group';
+export const LOCATION_NAME = 'my_location';
+export const AMP_SENSOR_ID = 'default_automation_sensor';
+
+export const assetGroupWithoutAMP: MockedResponse<
+  AssetGroupMetadataQuery,
+  AssetGroupMetadataQueryVariables
+> = {
+  request: {
+    query: ASSET_GROUP_METADATA_QUERY,
+    variables: {
+      selector: {
+        groupName: GROUP_NAME,
+        repositoryName: DUNDER_REPO_NAME,
+        repositoryLocationName: LOCATION_NAME,
+      },
+    },
+  },
+  result: {
+    data: {
+      __typename: 'Query',
+      assetNodes: [
+        buildAssetNode({
+          id: 'node_vanilla',
+          autoMaterializePolicy: null,
+          automationPolicySensor: null,
+        }),
+      ],
+    },
+  },
+};
+
+export const assetGroupWithAMP: MockedResponse<
+  AssetGroupMetadataQuery,
+  AssetGroupMetadataQueryVariables
+> = {
+  request: {
+    query: ASSET_GROUP_METADATA_QUERY,
+    variables: {
+      selector: {
+        groupName: GROUP_NAME,
+        repositoryName: DUNDER_REPO_NAME,
+        repositoryLocationName: LOCATION_NAME,
+      },
+    },
+  },
+  result: {
+    data: {
+      __typename: 'Query',
```

Comment:
You should try `buildQueryMock` allows you to skip typing `__typename: Query` every time and slightly less boilerplate for mocking queries.